spec/ruby/core/string/index_spec.rb

```diff
@@ -260,6 +260,7 @@
     "blablabla".index(/.{10}/).should == nil
     "blaxbla".index(/.x/, 3).should == nil
     "blaxbla".index(/..x/, 2).should == nil
+    "I’ve got a multibyte character.\n".index "\n\n"
```

Comment:
Is there an expectation or matcher for this? https://github.com/ruby/spec/blob/master/CONTRIBUTING.md#matchers-and-expectations

Reply:
It's missing an expectation entirely here.

Reply:
Oops. The failure was that it looped forever, so I hadn't noticed I'd missed the matcher entirely. :grin: